client/components/Layout/LayoutPagesCollections.tsx

```diff
@@ -37,6 +37,7 @@ const resolveItemsFromContent = (
 
 const LayoutPagesCollections = (props: Props) => {
 	const { content, collections, pages } = props;
+	const gridTemplateColumnsCSSFill = content.justify === 'center' ? 'auto-fit' : 'auto-fill';
```

Comment:
Initialisms in camel-cased variable names in our codebase tend not to have all letters capitalized, so we'd more likely write `gridTemplateColumnsCssFill`.

Reply:
changed it